packages/rules/lib/rules/prefer-orbit-components.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const BannedComponents = ["Button", "Label"]
```

Comment:
Input will also be available in Orbit soon? You might want to wait before merging the PR, your call

Reply:
I'll update the rule as new components are released.